### ISSUE TITLE
NullInjectorError: No Provider for e! only on build --prod

### ISSUE THREAD

**Issue by JonnJJordan** [Innqube/angular-calendar-week-hours-view#8]
Angular v5.2.10
Angular-calendar-week-hours-view v1.0.1

```
ERROR Error: StaticInjectorError(he)[n -> e]: 
  StaticInjectorError(Platform: core)[n -> e]: 
    NullInjectorError: No provider for e!
    at t.get (main.0588cac0b8475445b9a3.bundle.js:1)
    at main.0588cac0b8475445b9a3.bundle.js:1
    at t (main.0588cac0b8475445b9a3.bundle.js:1)
    at t.get (main.0588cac0b8475445b9a3.bundle.js:1)
    at main.0588cac0b8475445b9a3.bundle.js:1
    at t (main.0588cac0b8475445b9a3.bundle.js:1)
    at t.get (main.0588cac0b8475445b9a3.bundle.js:1)
    at Ii (main.0588cac0b8475445b9a3.bundle.js:1)
    at t.get (main.0588cac0b8475445b9a3.bundle.js:1)
    at Ii (main.0588cac0b8475445b9a3.bundle.js:1)
```

This only happens when I execute `ng build --prod` or `ng serve --prod` as well.

**Comment by sebastianmusial** [comment]
@JonnJJordan @diegofsza Did you found solution for this problem?

**Comment by diegofsza** [comment]
Not yet. The missing provider is CalendarDateFormatter. Will try to look for a solution in the following days.

**Comment by diegofsza** [comment]
Fixed with version 1.0.2
